Makefile

```
@@ -186,19 +231,86 @@ commands: ## List all Symfony commands
 	tests/Load/aws-execute-load-tests.sh
 
 aws-load-tests-cleanup: ## Cleanup AWS infrastructure after testing
 	tests/Load/cleanup.sh
+
+start-prod-loadtest: ## Start production environment with load testing capabilities
+	$(DOCKER_COMPOSE) -f docker-compose.loadtest.yml up --detach
+
+stop-prod-loadtest: ## Stop production load testing environment
+	$(DOCKER_COMPOSE) -f docker-compose.loadtest.yml down --remove-orphans
+
+ci: ## Run comprehensive CI checks (excludes bats and load tests)
```

checkmake (0.2.2)

[warning] Target body for "ci" exceeds allowed length of 5 (36).

(maxbodylength)